autoreject/utils.py

```diff
@@ -454,8 +446,9 @@ def _fast_map_meg_channels(info, pick_from, pick_to,
                origin=my_origin, noise=noise, self_dots=self_dots,
                surface_dots=cross_dots, int_rad=int_rad, miss=miss)
 
-    fmd['data'] = _compute_mapping_matrix(fmd, info_from)
-    mne.set_log_level(verbose)
+    # XXX: hack to silence _compute_mapping_matrix
+    with mne.utils.use_log_level('error'):
```

Comment:
`'error'` or `verbose` ?

Reply:
'error' is fine here -- the only arg to `use_log_level` is `verbose` so it's implicitly `verbose='error'` (but in practice always written as `use_log_level('info')` etc. in MNE)